environment/prod/deployment/prod/.env.prod

```diff
@@ -0,0 +1,29 @@
+COMPOSE_PROJECT_NAME=blumilk-website-prod
+
+BLUMILK_WEBSITE_HOST_NAME=blumilk.test.pl
```

Comment:
Should be changed. 
We can use temporary domain for test purposes like `prod.website.blumilk.pl` (this need to be added to the DNS provider), then migrate data and check. If everything will be fine, we will only switch domains.